packages/opentelemetry-host-metrics/src/stats/si.ts

```diff
@@ -14,12 +14,12 @@
  * limitations under the License.
  */
 
-import * as SI from 'systeminformation';
+import { networkStats } from 'systeminformation/lib/network';
```

Comment:
I'm not sure about `systeminformation` but deep-imports like this are usually not covered by stability guarantees. would this work for you?
```suggestion
import { networkStats } from 'systeminformation';
```

Reply:
Yes

Reply:
Sadly not, it still imports the cpu.js file in the package. Resulting in having to add the unnecessary dependency `osx-temperature-sensor`, which is not even used by networkStats

Reply:
I'm resolving this conversation. It was discussed in separate comments on this PR and in the SIG call recently.